lib/plugin.js

```diff
@@ -104,56 +128,60 @@ class VueLoaderPlugin {
   }
 }
 
-function createMatcher (fakeFile) {
-  return (rule, i) => {
-    // #1201 we need to skip the `include` check when locating the vue rule
-    const clone = Object.assign({}, rule)
-    delete clone.include
-    const normalized = RuleSet.normalizeRule(clone, {}, '')
-    return (
-      !rule.enforce &&
-      normalized.resource &&
-      normalized.resource(fakeFile)
-    )
+function cloneRule (rawRule, ruleSetCompiler, refs) {
```

Comment:
There's only one instance of `ruleSetCompiler` in this module so no need to pass it around.